bioblend/_tests/TestGalaxyWorkflows.py

```diff
@@ -205,15 +205,88 @@ def test_run_workflow(self):
             self.gi.workflows.run_workflow(wf['id'], None)
 
     def test_invoke_workflow(self):
+        invoke_response = self._invoke_workflow()
+        assert invoke_response['state'] == 'new', invoke_response
+
+    def test_create_workflow_from_history(self):
+        path = test_util.get_abspath(os.path.join('data', 'paste_columns.ga'))
+        wf1 = self.gi.workflows.import_workflow_from_local_path(path)
+        history_id = self.gi.histories.create_history(name="test_wf")['id']
+        dataset_id = self._test_dataset(history_id)
+        dataset = {'src': 'hda', 'id': dataset_id}
+
+        _job = sorted(self.gi.jobs.get_jobs(), key=lambda x: x['create_time'], reverse=True)[0]
+
+        self.gi.workflows.invoke_workflow(
+            wf1['id'],
+            inputs={'Input 1': dataset, 'Input 2': dataset},
+            history_id=history_id,
+            inputs_by='name',
+        )
+
+        time.sleep(5)
+        job = sorted(self.gi.jobs.get_jobs(), key=lambda x: x['create_time'], reverse=True)[0]
```

Comment:
```suggestion
        job = sorted(self.gi.jobs.get_jobs(), key=lambda x: x['create_time'], reverse=True)[0]
        job_id = gi.invocations.show_invocation(inv_id)['steps'][-1]['job_id']
```
Can you do something like this?

Reply:
`self.gi.invocations.show_invocation` is only availabe in Galaxy 19.09+ so tests would fail for all older versions.

Reply:
Ah, got it. Then this is an okay solution. It would be nice if get_jobs allowed filtering, but that's not something for this PR.